lib/movie/screens/movie_details_screen/movie_details_screen.dart

```diff
@@ -0,0 +1,317 @@
+          ),
+        );
+      },
+    );
+  }
+}
+
+class InfoRow extends StatelessWidget {
+  const InfoRow({
+    Key? key,
+    required this.posterPath,
+    required this.voteAverage,
+    required this.isAdult,
+    required this.releaseDate,
+  }) : super(key: key);
+  final String posterPath;
+  final String voteAverage;
+  final String isAdult;
+  final String releaseDate;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        Poster(posterPath: posterPath),
+        const SizedBox(width: _kWidth),
+        InfoText(
+          voteAverage: voteAverage,
+          isAdult: isAdult,
+          releaseDate: releaseDate,
+        ),
+      ],
+    );
+  }
+}
+
+class BackgroundImage extends StatelessWidget {
+  const BackgroundImage({
+    Key? key,
+    required this.posterPath,
+  }) : super(key: key);
+
+  final String posterPath;
+  @override
+  Widget build(BuildContext context) {
+    return Ink.image(
+      image: NetworkImage(posterPath),
+      fit: BoxFit.cover,
+      child: BackdropFilter(
+        filter: ImageFilter.blur(sigmaX: 10.0, sigmaY: 10.0),
+        child: Container(
+          decoration: BoxDecoration(
+            color: Colors.black.withOpacity(0.8),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class Poster extends StatelessWidget {
+  const Poster({
+    Key? key,
+    required this.posterPath,
+  }) : super(key: key);
+
+  final String posterPath;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: 280,
+      width: 200,
+      decoration: BoxDecoration(
+        border: Border.all(),
+        borderRadius: BorderRadius.circular(20),
+        image: DecorationImage(
+          fit: BoxFit.cover,
+          image: NetworkImage(posterPath),
+        ),
+      ),
+    );
+  }
+}
+
+class InfoText extends StatelessWidget {
+  const InfoText({
+    Key? key,
+    required this.voteAverage,
+    required this.isAdult,
+    required this.releaseDate,
+  }) : super(key: key);
+  final String voteAverage;
+  final String isAdult;
+  final String releaseDate;
+
+  @override
+  Widget build(BuildContext context) {
+    return Expanded(
+      child: SizedBox(
+        height: 280,
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            _Text(
+              title: _kScore,
+              text: voteAverage,
+            ),
+            _Text(
+              title: _kRating,
+              text: isAdult,
+            ),
+            _Text(
+              title: _kReleaseDate,
+              text: releaseDate,
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _Text extends StatelessWidget {
+  const _Text({
+    Key? key,
+    required this.title,
+    required this.text,
```

Comment:
it would be more correct to call this parameter **subtitle**
replace `text` to `subtitle`